### ISSUE TITLE
💡[feat] delete batch of experiment and checkpoints

### ISSUE THREAD

**Issue by ghstgs** [determined-ai/determined#6554]
### Describe the problem

deleting experiments/checkpoints one by one takes a while

### Describe the solution you'd like

could we delete multiple experiments with

yes| det e delete 1 2 3 4 5

instead of having to do
yes| det e delete 1
yes| det e delete 2
yes| det e delete 3
yes| det e delete 4
yes| det e delete 5

it would also be nice to be able to delete all checkpoints from an experiment (while keeping the logs) rather than havin to de e lc and then delete them one by one...

det e delete-all-checkpoints EXP-ID

**Comment by rb-determined-ai** [comment]
yeah, this is a good idea.  I actually want to see this pattern all over our CLI and SDK (cc @wes-turner).

**Comment by rb-determined-ai** [comment]
Until we can implement this feature, I can also say that doing the deletion in the python sdk, in a for loop, is much faster than running the cli in a for loop, if you're willing to write some python code.
